hal/armv7a/imx6ull/_init.S

```diff
@@ -88,6 +88,17 @@ plugin:
 	/* note: "ldr reg, =constant" pseudo instruction can't be used before
 	 * the serial download check below - the literal pool will not be included in the plugin */
 
+	/* init watchdog */
+	mov32 r4, 0x20bc000
+#if defined(WATCHDOG)
+	/* 128 sec timeout, the clock is already enabled */
+	movw r5, #0xff34
+#else
+	/* if watchdog is disabled feed him default value */
```

Comment:
Maybe add comment that it's needed to keep `plugin` section size the same regardless of `WATCHDOG` define?

Reply:
Good idea, it might get removed if no rationale is presented